tests/containers/python/Dockerfile

```diff
@@ -1,6 +1,8 @@
-FROM python:3.6-slim
+FROM python:3.6.15-slim
 
 WORKDIR /app
 ADD lister.py /app
+# Install some package so we can test that its info appears in the collapsed
+RUN pip install setuptools==59.4.0
```

Comment:
I think it'll be better to install something that will be less confusing, maybe some parsing library or something that doesn't do IO/networking

Reply:
How is this confusing?

Reply:
because we're testing packages info gathering on package that interact with packages.

Reply:
Oh OK 😅 
I choose this package because it contains a module with a different name in it, which supposed to make the test somewhat harder.

Reply:
Changed to `PyYAML` instead

Reply:
noice